deployment/terraform/modules/docker_host/main.tf

```diff
@@ -0,0 +1,57 @@
+variable "cidr_block" {
+  type = "string"
+  default = "10.0.0.0/16"
+}
+
+variable "security_group_id" {
+  type = "string"
+  default = "my_security_group_id"
```

Comment:
I'd simply assign a value in the corresponding `module` section of `live/main.tf` based on `${terraform.workspace}` ([example](https://github.com/refinery-platform/refinery-platform/blob/ff1167230df226f27f932f846452cdbd8f901dd0/deployment/terraform/live/main.tf#L25))